### PR TITLE
#OLS-451 added woke rules and fixed inclusive-naming incompatabilities

### PR DESCRIPTION
**Pull request by nemerna**
## Description

Added .woke.yaml config file that aligns with https://inclusivenaming.org/ requirements
scanned the project with woke-cli and fixed the inclusive-naming incompatibilities

more info on woke-cli: https://github.com/get-woke/woke

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue #
- Closes #[OLS-451](https://issues.redhat.com//browse/OLS-451)

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

**Comment by nemerna** [comment]
i have added woke-cli to the install-tools Makefile Rule
and added woke checks to be in the verify rule

File: ols/utils/keywords.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "k8s",
     "kubernetes",
     "log",
-    "master",
+    "control-plane",
```
[the remark this answers is not in the document]
Comment (bparees) [reply ols/utils/keywords.py:26]: let's exclude the file then

File: Makefile [hunk 2]
```diff
@@ -13,9 +13,12 @@ SCENARIO := $(if $(SCENARIO),$(SCENARIO),"with_rag")
 images: ## Build container images
 	scripts/build-container.sh
 
-install-tools: ## Install required utilities/tools
+install-tools:	install-woke ## Install required utilities/tools
 	@command -v pdm > /dev/null || { echo >&2 "pdm is not installed. Installing..."; pip install pdm; }
 
+install-woke: ## Install woke, required for Inclusive Naming scan
+	@command -v ./woke > /dev/null || { echo >&2 "woke is not installed. Installing..."; curl -sSfL https://git.io/getwoke | bash -s -- -b ./; }
```
Comment (bparees) [inline Makefile:20]: did you add `woke` to your .gitignore?

Comment (nemerna) [reply Makefile:20]: nice catch
it was added locally but forgot to include it in the commit

**Pull request merged**